projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -382,6 +386,16 @@ function ( $preference ) {
 		die();
 	}
 
+	/**
+	 * Handle ajax requests to dismiss a just-in-time-message
+	 */
+	public function wp_ajax_jitm_dismiss() {
+		check_ajax_referer( 'jitm_dismiss' );
+		require_lib( 'jetpack-jitm/jitm-engine' );
```

Comment:
Nitpick: There is some discussion about whether `require_lib` should be used or not (see p7H4VZ-2QA-p2). I personally prefer using `require_once` because it looks more readable and direct to me.